### ISSUE TITLE
Usage of scaleway_instance_security_group_rules  triggers constant terraform change. 

### ISSUE THREAD

**Issue by alekc** [terraform-providers/terraform-provider-scaleway#445]
### Terraform Configuration Files
```hcl
resource "scaleway_instance_security_group" "test-sg" {
  external_rules = true
}

resource scaleway_instance_security_group_rules sgrs01 {
  security_group_id = scaleway_instance_security_group.test-sg.id
  inbound_rule {
    action = "accept"
    port = 80
    ip = "1.1.1.1"
  }
  inbound_rule {
    action = "accept"
    port = 81
    ip = "2.2.2.2"
  }
}
```

### Expected Behavior
Once applied hcl above, I expect for `terraform plan` to say 0 changes 0 deletion

### Actual Behavior
```
# scaleway_instance_security_group.test-sg will be updated in-place
  ~ resource "scaleway_instance_security_group" "test-sg" {
        external_rules          = true
        id                      = "fr-par-1/e4c7b0fb-ec15-47b6-ac09-26359c1406fe"
        inbound_default_policy  = "accept"
        name                    = "tf-sg-lucid-antonelli"
        organization_id         = "df378afc-8280-4a58-b486-1a1247e4798a"
        outbound_default_policy = "accept"
        zone                    = "fr-par-1"

      - inbound_rule {
          - action     = "accept" -> null
          - ip_range   = "1.1.1.1/32" -> null
          - port       = 0 -> null
          - port_range = "80-0" -> null
          - protocol   = "TCP" -> null
        }
      - inbound_rule {
          - action     = "accept" -> null
          - ip_range   = "2.2.2.2/32" -> null
          - port       = 0 -> null
          - port_range = "81-0" -> null
          - protocol   = "TCP" -> null
        }
    }
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`
2. `terraform plan`

It looks like we are not checking if a security group has the flag external_rules set to true, so we populate it's rules on every load.

**Comment by Sh4d1** [comment]
Hey @alekc ! Which version of TF provider ?

**Comment by Sh4d1** [comment]
@alekc fixed by #446 :smile:
